apps/site/assets/css/_stop-page.scss

```diff
@@ -317,6 +317,38 @@
     @include media-breakpoint-down(sm) {
       margin-bottom: 1rem;
     }
+
+    &[disabled] {
+      background-color: $gray-lightest;
+      border-color: $gray-light;
+      color: $body-color;
+
+      // override set .c-descriptive-link title
+      .c-descriptive-link__title,
+      // override SVG components that have a hard-coded fill
+      [class*='c-svg__icon'] :where(path, rect)[fill] {
+        color: inherit;
+        fill: currentColor;
+      }
+    }
+  }
+
+  .c-descriptive-link__header {
```

Comment:
what does `c` stand for?  I assumed `m` stood for mobile, but I could be wrong with that too

Reply:
A relic of the past and I don't remember where the doc describing it is, but c is for component and m is for module.